pages/standards-guidelines/wcag/wcag3-intro.md

```diff
@@ -92,36 +85,20 @@ Each normative section has a status that indicates how far along in the developm
 * Refining
 * Mature
 
-For more information on these status indicators, see the Explainer section [Current Process for Creating WCAG 3.0](https://www.w3.org/TR/2024/DNOTE-wcag-3.0-explainer-20241212/#current-process).
-
-### For your review
-
-In this WCAG 3 Working Draft, there are four updated sections that we would particularly like feedback on. The first three sections are guidelines:
-
-1. [Image alternatives guideline](https://w3.org/TR/wcag-3.0/#image-alternatives)
-2. [Keyboard focus appearance guideline](https://w3.org/TR/wcag-3.0/#keyboard-focus-appearance)
-3. [Clear meaning guideline](https://w3.org/TR/wcag-3.0/#clear-meaning)
+For more information on these status indicators, see the Explainer section [Current Process for Creating WCAG 3.0](https://www.w3.org/TR/wcag-3.0-explainer/#current-process).
 
-For these sections, does the information under 'Which foundational requirements apply?' make the guidelines easier to understand and use?
+### September 2025 update
 
-The other section we would like feedback on is:
+This update includes potential guidelines, requirements, and assertions that have progressed to Developing status.
```

Comment:
Please add something like: "Guidelines, requirements and assertions that are still Exploratory are available in the Editor's Draft." with link.

Reply:
+1

I changed it to: "This update includes potential guidelines, requirements, and assertions that have progressed to Developing status. Those that are still Exploratory are available in the _Editor's Draft_." I think not repeating "guidelines, requirements, and assertions" is a bit easier to process. Feel free to re-open/comment if you suggest otherwise.